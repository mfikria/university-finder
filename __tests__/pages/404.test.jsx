/**
 * @jest-environment jsdom
 */

import React from 'react'
import { render } from '@testing-library/react'
import NotFound from '../../pages/404'

describe('NotFound', () => {

  it('should match snapshot', () => {
    const { asFragment } = render(<NotFound />)
    expect(asFragment()).toMatchSnapshot()
  })
})
