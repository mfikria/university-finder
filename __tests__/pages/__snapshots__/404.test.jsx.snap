// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="layout404"
  >
    <div
      class="title"
    >
      404
    </div>
    <h1
      class="ant-typography"
    >
      Oopss... Page Not Found
    </h1>
    <button
      class="ant-btn"
      type="button"
    >
      <a
        href="/"
      >
        Back to home
      </a>
    </button>
  </div>
</DocumentFragment>
`;
